### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rhagarty**
My attempt to implement suggestions by Madison

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,12 +38,12 @@ Follow these steps to setup and run this developer journey. These steps are desc
...
+Sign up for IBM's [Data Science Experience](http://datascience.ibm.com/). By signing up for the Data Science Experience, two services will be created in your Bluemix account: ``DSX-Spark`` and ``DSX-ObjectStore``. If these services do not exist, or if you are already using them for some other application, you will need to create new instances.
 
 To create these services:
-* Login to your [Bluemix](http://bluemix.net) account.
-* Create your Spark service by selecting the service type [Apache Spark](https://console.bluemix.net/catalog/services/apache-spark). If not already used, name your service ``DSX-Spark``. 
-* Create your Object Storage service by selecting the service type [Cloud Object Storage](https://console.bluemix.net/catalog/infrastructure/object-storage-group). If not already used, name your service ``DSX-ObjectStorage``.
+* Login or create to your [Bluemix](http://bluemix.net) account.
```
Comment (stevemar) [inline README.md]: this doesn't make sense now: "Login or create to your Bluemix"

**Review by stevemar — APPROVED** [review]
only one minor comment. looks good otherwise

**Comment by stevemar** [comment]
I reviewed and left a +A, happy to give @asurve a chance to bless the patch.

**Comment by asurve** [comment]
Thanks @rhagarty & @stevemart LGTM

**Comment by rhagarty** [comment]
Would like feedback from @MadisonJMyers

**Review by MadisonJMyers — APPROVED** [review]
LGTM thanks for implementing those!

**Pull request merged**